tests/_utils.py

```diff
@@ -42,3 +42,16 @@ def generate_plain_records(count: int = 100) -> Iterator[Record]:
 
     for i in range(count):
         yield TestRecord(number=i, dt=datetime.datetime.now(datetime.timezone.utc))
+
+
+def generate_command_records(count: int = 100) -> Iterator[Record]:
+    TestRecord = RecordDescriptor(
+        "test/addapter/command",
+        [
+            ("uint32", "number"),
+            ("command", "data"),
+        ],
+    )
+
+    for i in range(count):
+        yield TestRecord(number=i, data=f"/path/to/file {i}")
```

Comment:
I'm not sure if we really need a specific generator for one specific field type. Also, it doesn't generate windows commands.

I think it would be better to just create specific TestRecord in your tests and with some specific real life example commands.